uwblueprint/focus-on-nature
# feat: create camper model

## Notion ticket link
[Camper Model](https://www.notion.so/uwblueprintexecs/Task-Board-38b49f83f6ce4b07ae188dabb37631e8?p=eeef6ad993534528947858dbaa3517c4)



## Implementation description
* added a Camper model in camper.model.ts
* added the CamperDTO in types.ts
* defined any new types for the model in types.ts
* defined a test function in index.ts to see if an entry gets created in mongodb



## Steps to test
1. Edit the camperModel.create function with different values in index.ts
2. Build the application
3. Go into mongodbb -> cluster0 -> collections -> campers to check that the entry has been made
Note: should work with empty string for optional fields


## What should reviewers focus on?
* Is this everything that needs to be done for the model?


## Checklist
- [x ] My PR name is descriptive and in imperative tense
- [x ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
